gatherers/service-worker.js

```diff
@@ -19,19 +19,25 @@
 const Gather = require('./gather');
 
 class ServiceWorker extends Gather {
+  setup(options) {
```

Comment:
hmm I will move it into here for now, I had it in the super class because I wanted to extract it for reuse, but the argument is invalid until someone else needs it.